src/server/worldserver/worldserver.conf.dist

```diff
@@ -3525,7 +3525,23 @@ LFG.Location.All = 0
 #         2 = Display detailed requirements, all at once, with clickable links
 #        Default: 1
 #  
-DungeonAccessRequirementsPrintMode = 1
+DungeonAccessRequirements.PrintMode = 1
+
+#    DungeonAccessRequirement.PortalAvgIlevelCheck
+#
+#    	 Description: Allow average ilvl to be checked when entering via portal and deny the entry if minimum average is not met.
+#        Default:     0 - (Disabled)
```

Comment:
#        Description: Enable average item level requirement when entering a dungeon/raid's portal (= deny the entry if player has too low average ilevel, like in LFG).
#        Default:     0 - (Disabled - Blizzlike)